data/json/morale_types.json

```diff
@@ -124,11 +124,6 @@
     "type": "morale_type",
     "text": "Ate human flesh"
   },
-  {
-    "id": "morale_demicannibal",
```

Comment:
If you don't mind leaving this in and making a bug report and I'll assign it to myself